snowpack/src/build/build-pipeline.ts

```diff
@@ -181,7 +181,7 @@ async function runPipelineTransformStep(
           isDev,
           isPackage,
           fileExt: destExt,
-          id: filePath,
```

Comment:
However, changing this seems preferable, as it is merely a reference to the original source file.

When we first released the plugin API, we had some discussion and decided to pass in a faux filepath for transform plugins. This was just to indicate to transform plugins that the source may have changed, but IMO that’s a given. Because of this PostCSS issue, I’m changing my mind and I think it’s better to give the original source file to transform plugins. It’s not only more useful; it also makes them more aligned with `load()` plugins and IMO more predictable to use.

Reply:
I'm on board with this reasoning!